pkg/assembler/clients/operations/delete.graphql

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright 2024 The GUAC Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This is experimental and might change in the future!
+
+# Delete nodes based on ID (and all associated edges)
+# Deletion is only implemented for HasSBOM, HasSLSA and CertifyVuln
+# for the time being. Other may be added based on usecase but these
+# were chosen to ensure that users do not end up making breaking changes
+# to their database.
+
+mutation Delete($nodeID: ID!) {
```

Comment:
Currently this is not being used. Need to decide the best flow to follow for deletion